components/logins/src/sync/engine.rs

```diff
@@ -640,8 +676,8 @@ mod tests {
                 (
                     sync15::Payload::from_json(serde_json::json!({
                         "id": "dummy_000003",
-                        "formActionOrigin": "https://www.example.com/submit",
```

Comment:
This also fixes a regression on this branch where the names of the items in the payload were changed from `hostname` to `origin` and `formSubmitURL` to `formActionOrigin` - the names in the payload must match other clients.